mem-04-gh.xml

```diff
@@ -1093,12 +1093,13 @@ The object of this verb is the question asked.</column>
       <column name="entry_name">ghe'chem</column>
       <column name="part_of_speech">n</column>
       <column name="definition">terraforming field</column>
-      <column name="definition_de">Terraformierungfeld</column>
+      <column name="definition_de">Terraforming-Feld</column>
       <column name="synonyms"></column>
       <column name="antonyms"></column>
       <column name="see_also"></column>
       <column name="notes">This is a short way to refer to an energy field of some sort that causes terraforming to happen. A longer way to refer to this concept would be {ghe'moHbogh HoSchem:n@@ghe':v, -moH:v, -bogh:v, HoSchem:n}.</column>
       <column name="notes_de"></column>
+      <!-- In German version of Netflix, it is translated as Terraforming-Bereich, which refers to an area. So maybe Stamets was not talking about a technical field, but just to a field of land -->
```

Comment:
But who translated the English into German for Netflix? Maybe they just didn't understand the concept?